### DISCUSSION TITLE
Ruta donde debo de colocar el html de pasteleria en vuex

### DISCUSSION THREAD

**Discussion by AlanGomezSK** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#544] [Q&A]
Tengo una duda, ya que no se en que ruta colocar el html de la pasteleria y del pastelero, acaso debo de editar el HelloWorld.vue o crear archivos con la extencion .vue en vez de html en la carpeta de componets
![Captura de pantalla 2022-03-27 142138](https://user-images.githubusercontent.com/99113073/160299379-20b5747c-07f5-42ac-853a-d6476664d2f0.png)
?

**Comment by bryan-vg** [comment]
yo te recomiendo ver por lo menos el ultimo live de vue, donde explican todo lo de los componentes y como incluirlos en las vistas
segun entendi yo, el html de cada parte de la pagina (pasteleria, info para el pastelero, etc) va en un componente de vue
![componentes de vue 2022-03-27 16-08-39](https://user-images.githubusercontent.com/99158978/160303085-d0b6eecd-e882-463d-aa0d-d47cd0f198a5.png)

y en una vista de vue es donde se incluye cada componente que se quiera mostar en cada pantalla. 
![vistas de vue 2022-03-27 16-07-53](https://user-images.githubusercontent.com/99158978/160303102-0ce413b9-3473-43e8-bc8f-e872b37cd7c7.png)

Pero como dije, queda mucho mas claro viendo el ultimo live de vue.

**Reply by AlanGomezSK** [reply]
Perfecto esa era toda mi duda, muchas gracias!!!

**Comment by doguedogue** [comment]
Si te refieres a la extensión HTML, no hay más que el Index.

Los demás componentes son como el HelloWorld.vue, dentro traen el template de código HTML.

Las demás vistas van en views.

Fíjate en este repositorio de ejemplo del MC

https://github.com/romarpla/FrontEnd-Mision/tree/main/05%20-%20VUEJS/programas/VUEX

**Reply by AlanGomezSK** [reply]
Va, muchas gracias!!

**Reply by gersondlarosa** [reply]
Duda a ti si te genero la liga para poderlo mostrar en la geb de git como liga

**Comment by OmarVenturaP** [comment]
De hecho el index.html ya lo genera por defecto en la carpeta públic. Allí le puedes agregar los scripts y los links que necesites, además de modificar el título.
